src/plugins/vis_type_xy/public/editor/components/options/point_series/point_series.tsx

```diff
@@ -54,7 +59,7 @@ export function PointSeriesOptions(
         </EuiTitle>
         <EuiSpacer size="m" />
 
-        <BasicOptions {...props} />
+        <BasicOptions {...props} collections={collections} />
```

Comment:
what if pass only `legendPositions`? That change will simplify code a little

`<BasicOptions {...props} legendPositions={legendPositions} />`

Reply:
agree

Reply:
done